contracts/test/strategies/generalized-meta.fork-test.js

```diff
@@ -218,16 +218,24 @@ metastrategies.forEach(
 
             await fixture.metaStrategy
               .connect(sGovernor)
-              .setMaxWithdrawalSlippage(ousdUnits("0.001"));
+              .setMaxWithdrawalSlippage(ousdUnits("0"));
             await tiltToMainToken(fixture);
 
-            await expect(
-              vault
+            let error = false;
```

Comment:
Is there a reason for this longer form of catching a revert and checking the error message than using the hardhat helper, `to.be.revertedWith()`?


https://github.com/OriginProtocol/origin-dollar/blob/1bb1a410c364149980af815b536ef89328c01df6/contracts/test/vault/index.js#L57-L59

Reply:
Yeah, when slippage errors occur different StableSwap contract implementations fail with different error messages. For `alUsd`, `USDD` and `BUSD` transaction fails without an error message, and `mUSD` fails with `VM Exception while processing transaction: reverted with reason string 'Not enough coins removed' `

So wanted to cover both cases and didn't find `chai` having some sort of expect.to.fail.with.**or** functionality

Reply:
Makes sense! Maybe a comment to explain why we are doing it differently.

Reply:
good point adding that